### DISCUSSION TITLE
Add a command to open emoji picker

### DISCUSSION THREAD

**Discussion by kosssi** [outline/outline#7643] [Ideas]
Maybe `/emoji` should open an emoji picker?

**Comment by tommoor** [comment]
`:` opens an emoji picker ([docs](https://docs.getoutline.com/s/guide/doc/emoji-cRBnDOYndG))

**Reply by kosssi** [reply]
Yes, but when users are uncomfortable with English or the names of emojis, he prefers to have a picker ;)

That's the third user asking me the question, that's why I created this discussion.
